frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/common/ErrorMessages.java

```diff
@@ -37,13 +37,14 @@
     public static final String EXTERNAL_ID_CANNOT_BE_EMPTY = "External ID cannot be empty.";
     public static final String PASSWORD_CANNOT_BE_EMPTY = "Password cannot be empty.";
     public static final String PASSWORDS_DONT_MATCH = "Password do not match.";
-    public static final String COULD_NOT_CREATE_USER_MODERATION_REQUEST = "Could not create user moderation request";
+    public static final String COULD_NOT_CREATE_USER_MODERATION_REQUEST = "Could not create user moderation request.";
     public static final String EMAIL_ALREADY_EXISTS = "Email already exists.";
     public static final String FULL_NAME_ALREADY_EXISTS = "Full name already exists.";
     public static final String EXTERNAL_ID_ALREADY_EXISTS = "External id already exists.";
     public static final String DEFAULT_ERROR_MESSAGE = "Request could not be processed.";
     public static final String DOCUMENT_NOT_AVAILABLE = "The requested document is not available.";
     public static final String LICENSE_SHORTNAME_TAKEN = "License shortname is already taken.";
+    public static final String UPDATE_FAILED_SANITY_CHECK = "Document update has not been performed, as it failed the sanity check.";
```

Comment:
I think the normal user does not know what "the sanity check" is. We should tell the user explicitly that he/she cannot delete ALL linked releases or projects at once, if the there are more than 5 of them.

Reply:
yes, maybe mention that "too many items (>5) were attemped to delete at once".